.github/workflows/golangci-lint.yml

```diff
@@ -59,5 +59,5 @@ jobs:
           # Don't forget to update
           # docs/Contributing/Testing-and-local-development.md when this
```

Comment:
Please update:
https://github.com/fleetdm/fleet/blob/9d99b7fb05cc6fd0b7f25d71b02f6b87de6a05b5/docs/Contributing/Testing-and-local-development.md?plain=1#L71